### PR TITLE
Test1

### PR DESCRIPTION
**Pull request by esirK**

### CODE REVIEW

File: Classes/Dojo/Dojo_main.py [hunk 1]
```diff
@@ -0,0 +1,223 @@
...
+
+from Person.person_new import Person
+from Person.Staff.staff import Staff
+from rooms.room_new import Room
+import random
+import collections
+
+
+class Dojo(object):
+    def __init__(self):
```
Comment (mung4i) [inline Classes/Dojo/Dojo_main.py:13]: Use class variables for data structures. These structures will be initialized every time `class Dojo` runs.

File: Classes/Dojo/Dojo_main.py [hunk 2]
```diff
@@ -0,0 +1,223 @@
...
+                else:
+                    print ("No Living Room's Remaining")
+                    self.list_of_fellows_with_no_accommodation.append(person)
+
+        # Allocate room
+#
+
+    def create_room(self, name, office_or_cube):
+        office_or_cube = office_or_cube.upper()
+        classi = Room(name, office_or_cube)
```
Comment (mung4i) [inline Classes/Dojo/Dojo_main.py:181]: Read on the PEP8 style guide on how to name variables.

**Review by mung4i — APPROVED** [review]

**Review by LWanjiru — COMMENTED** [review]
Update the requirements.txt files whenever you update /install new packages

**Review by kimobrian — COMMENTED** [review]
Learn more about programming concepts like data structures, variable naming ,logic, OOP.

File: Classes/Dojo/rooms/Offices/office.py [hunk 3]
```diff
@@ -0,0 +1,3 @@
+class Office():
```
Comment (mung4i) [inline Classes/Dojo/rooms/Offices/office.py:1]: At the mvc stage of demo, you shouldn't have any classes defined as pass

**Review by mung4i — APPROVED** [review]

**Pull request closed without merging**